nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/header/new-canvas-item/new-canvas-item.component.scss

```diff
@@ -15,23 +15,24 @@
  * limitations under the License.
  */
 
-.icon {
-    font-size: 32px;
+.new-canvas-item {
```

Comment:
This is good because it contains the changes to the icon class here but are there other areas that were depending on these styles for any `.icon`'s?

Reply:
This aligned the style structure from the scss partial. Since `new-canvas-item` is the root element it should be functionally equivalent.

Reply:
OK makes sense and LGTM.